src/backend/codegen_gpu/extract_read_write_set.cpp

```diff
@@ -0,0 +1,36 @@
+#include "graphit/backend/codegen_gpu/extract_read_write_set.h"
+
+namespace graphit {
+void ExtractReadWriteSet::visit(mir::StmtBlock::Ptr stmt_block) {
+	return;
+}
+void ExtractReadWriteSet::visit(mir::TensorArrayReadExpr::Ptr tare) {
```

Comment:
I would use a more meaningful variable name, not "tare". In general, it would be easier if we don't use the acronyms.

Reply:
I agree, I have been a bit liberal with all the names here. I can take a pass and clean up all the variable names used in the GPU backend and the related passes.